Backpack/Snackbar/Classes/BPKKeyboardInfo.m

```diff
@@ -0,0 +1,73 @@
+/*
+* Backpack - Skyscanner's Design System
+*
+* Copyright 2018-2019 Skyscanner Ltd
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#import "BPKKeyboardInfo.h"
+#import "BPKSnackbar.h"
+
+@implementation BPKKeyboardInfo
+
++ (void)load
+{
+    [BPKKeyboardInfo sharedInstance];
+}
+
++ (instancetype)sharedInstance
+{
+    static BPKKeyboardInfo *sharedInstance;
+    static dispatch_once_t onceToken;
```

Comment:
Could we try having a single instance of `BPKKeyboardInfo` on the `BPKSnackBar` instance instead of having another singleton?